src/main/java/org/jabref/gui/util/KeyStrokeToString.java

```diff
@@ -0,0 +1,350 @@
+package org.jabref.gui.util;
+
+import java.awt.event.InputEvent;
+import java.awt.event.KeyEvent;
+import java.util.StringJoiner;
+
+import javax.swing.KeyStroke;
+
+/**
+ * Class code adapted from http://www.java2s.com/Tutorial/Java/0260__Swing-Event/ConvertingaKeyStroketoaString.htm
+ */
+public class KeyStrokeToString {
+
+    public static String keyStrokeToString(KeyStroke key) {
+        StringJoiner sj = new StringJoiner(" ");
```

Comment:
I hate to nitpick, but could you assign more meaningful names to this and the following variable?

Reply:
I googled for that and just added it. I think, it was always broken.

Reply:
This KeyStrokeToSTring class looks really odd. I am not sure if we really need it for javafx